examples/persistence-storage/compose.yml

```diff
@@ -0,0 +1,58 @@
+services:
+  db:
+    image: postgres:16-alpine
+    ports:
+      - ${DB_PORT}:5432
+    environment:
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_DB=${DB_DATABASE_NAME}
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -d ${DB_DATABASE_NAME} -U ${DB_USER}" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  db_migrator:
+    depends_on:
+      db:
+        condition: service_healthy
+    restart: "no"
+    image: ruurdadema/sqlx-cli
+    command: >
+      sh -c "sqlx database create &&
+             sqlx migrate run --source /migrations"
+    volumes:
+      - ../../migrations/postgres:/migrations
```

Comment:
Will this require the script to be launched from specific directory with migrations code checked out?

Reply:
Yes, it works without any problems -- the path is relative to `compose.yml` itself

```shell
➜  kamu-cli ✗ pwd
/home/s373r/src/work/kamu-data/kamu-cli

➜  kamu-cli ✗ docker compose -f ./images/persistent-storage/compose.yml up
[+] Running 3/0
 ✔ Container persistent-storage-db-1           Created                                                                                                                                                          
 ✔ Container persistent-storage-db_migrator-1  Created                                                                                                                                                          
 ✔ Container persistent-storage-kamu-1         Created  
...
db_migrator-1  | Applied 20240314140913/migrate accounts table (46.124515ms)
db_migrator-1  | Applied 20240328152300/migrate task events table (46.640685ms)
db_migrator-1  | Applied 20240422100311/migrate flow configuration events (39.125564ms)
...
 ```

Reply:
Not so friendly for UI devleopers, who do not even checkout CLI.
Let's think of a simpler mechanism that would somehow download these files into temp from the `kamu-cli` GIT.

Reply:
> Not so friendly for UI devleopers, who do not even checkout CLI.

good argument -- cloning a repository just for the sake of migrating migration files is a weakness of this PR -- will be corrected

Reply:
Done